crowbar_framework/app/models/ceph_service.rb

```diff
@@ -180,7 +190,28 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
         master.save
       end
     end
+  end
+
+  def apply_role_post_chef_call(old_role, role, all_nodes)
+    @logger.debug("ceph apply_role_post_chef_call: entering #{all_nodes.inspect}")
+    calamari = role.override_attributes["ceph"]["elements"]["ceph-calamari"] || []
 
+    calamari.each do |n|
+      node = NodeObject.find_node_by_name(n)
+      node.crowbar["crowbar"] ||= {}
+      node.crowbar["crowbar"]["links"] ||= {}
+
+      for t in ['admin'] do
+        unless node.get_network_by_type(t)
+          node.crowbar["crowbar"]["links"].delete("Calamari Dashboard (#{t})")
+          next
+        end
+        ip = node.get_network_by_type(t)["address"]
+        node.crowbar["crowbar"]["links"]["Calamari Dashboard (#{t})"] = "http://#{ip}/"
```

Comment:
Probably something for later, but this made me realize that the deployment is without SSL. I assume it's something easy to do by playing with the apache config?